spec/rubocop/cop/rake/desc_spec.rb

```diff
@@ -63,4 +63,17 @@
       task.name
     RUBY
   end
+
+  it 'does not register an offense for one prerequisite declaration (alias)' do
+    expect_no_offenses(<<~RUBY)
+      task release: 'changelog:check_clean'
+    RUBY
+  end
+
+  it 'registers an offense for multiple prerequisite declarations' do
+    expect_offense(<<~RUBY)
+      task release: ['changelog:check_clean', 'changelog:create']
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Describe the task with `desc` method.
```

Comment:
It seems to be consistent with the above changed behavior if no warning is registered this.

Reply:
My bad. Fixed now.